src/gui/editors/SongEditor.cpp

```diff
@@ -58,13 +58,17 @@ namespace lmms::gui
 {
 
 
-const QVector<float> SongEditor::m_zoomLevels =
+const QVector<float> SongEditor::m_zoomXLevels =
 		{ 0.125f, 0.25f, 0.5f, 1.0f, 2.0f, 4.0f, 8.0f, 16.0f };
 
+const QVector<float> SongEditor::m_zoomYLevels =
+		{ 1.0f, 1.25f, 1.5f, 1.75f, 2.0f, 2.5f, 3.0f, 4.0f, 8.0f, 16.0f };
```

Comment:
Any particular reason for these specific values? You may also want to look at #6664, which is changing the horizontal zoom control to a continuous slider.